maestrowf/__init__.py

```diff
@@ -264,6 +272,67 @@ def render_to_str(self, theme=None, width=200):
         return _printer.file.getvalue()
 
 
+class SummaryStatusRenderer(FlatStatusRenderer):
+    """Flat, simple summary table layout"""
+
+    layout_type = "summary"        # Defines name in factory/cli
+
+    def update_theme(self):
+        self._theme_dict["Step Prefix"] = "blue"
```

Comment:
I found this color scheme harder to read on a darker terminal, primarily on values that were a single character long. Might want to consider changing this up.

Reply:
I don't know what color scheme I was using previously, but this actually is coming over just fine to me.